src/pkg/task/sweep_manager.go

```diff
@@ -182,6 +185,32 @@ func (sm *sweepManager) Clean(ctx context.Context, execIDs []int64) error {
 	return nil
 }
 
+// FixDanglingStateExecution update executions always running
+func (sm *sweepManager) FixDanglingStateExecution(ctx context.Context) error {
+	ormer, err := orm.FromContext(ctx)
+	if err != nil {
+		return err
+	}
+
+	sql := `UPDATE execution
```

Comment:
This SQL only handles the execution status, but task seems also having some dangling status?

Reply:
task will be updated in src/jobservice/worker/cworker/reaper.go